src/Waiter.php

```diff
@@ -86,9 +88,12 @@ public function __construct(
         }
     }
 
+    /**
+     * @return Coroutine|PromiseInterface
```

Comment:
Shouldn't this always return a Coroutine?

Reply:
Yes, fixed it.